main.cpp

```diff
@@ -18,13 +18,21 @@ using namespace clang::ast_matchers;
 using namespace clang::tooling;
 
 class CastCallBack : public MatchFinder::MatchCallback {
+    Rewriter& _rewriter;
+
 public:
-    CastCallBack(Rewriter& rewriter) {
+    CastCallBack(Rewriter& rewriter): _rewriter(rewriter){
         // Your code goes here
+
     };
 
     void run(const MatchFinder::MatchResult &Result) override {
         // Your code goes here
+        const auto* styleCastExpr = Result.Nodes.getNodeAs<CStyleCastExpr>("cast");

```

Comment:
What if `styleCastExpr` is a null pointer? How to protect the program from crashing in such cases?

Reply:
Added nullptr protection
![image](https://user-images.githubusercontent.com/45869154/170296703-f75f40fb-cf25-4676-af19-5552abbd9170.png)